### DISCUSSION TITLE
Suggestion: enable open answers for oracles

### DISCUSSION THREAD

**Discussion by ubiubi18** [idena-network/idena-desktop#541] [Ideas]
Beside polls and certifying facts within  prediction markets ( will trump win election? Yes/No) and information gathering markets (did trump win election? Yes/No) and eventually combined with multiple choise/quadratic voting...

What about an "open oracle", that allows participants to give open answers sent in aa encrypted plain text in a first round and than upvote and downvote each others answers in a second encrypted round, publishing results in third round? 

That would enable to use idena as a more creative oracle of crowd wisdom in a way that is dependend on thoughtful input of real humans and can be a unique selling point with the potential to be used beyond blockchain sphere. There could be third party service providers, who collect questions against fiat money (so the questionäre has nothing to do with exchanges at all) and issue them on idenas oracles, buying up the neccessary iDNA themself

**Comment by midenaio** [comment]
Moving the issue to discussions
